style-guide.rst

```diff
@@ -520,32 +543,72 @@ A big advantage of reST in comparison to plain Markdown is that it allows to reu
 Substitution
 ~~~~~~~~~~~~
 
-To reuse sentences or paragraphs without too much markup and special formatting, use substitutions.
+To reuse sentences and entire paragraphs
+that have little markup or special formatting,
+define their `substitutions`_ in two possible ways.
+
+**Globally**, in a file named :file:`reuse/substitutions.txt`
+that is included in the `rst_epilog`_ directive:
+
+.. code-block:: python
+   :caption: :spellexception:`conf.py`
+
+   rst_epilog = """
+       .. include:: reuse/substitutions.txt
+       """
+
+
+.. code-block:: rest
+   :caption: :spellexception:`reuse/substitutions.txt`
+
+   .. |rest_text| replace:: *Multi-line* text
+                            that uses basic **markup**.
+
+   .. |site_link| replace:: Website link
+   .. _site_link: https://example.com
```

Comment:
Should we add an example for a version number, to show a typical use for substitutions?